PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/forms/AddNewClusterForm.java

```diff
@@ -62,9 +72,24 @@ public AddNewClusterForm(Shell parentShell, HDInsightRootModule module) {
     @Override
     protected void configureShell(Shell newShell) {
         super.configureShell(newShell);
+        newShell.setSize(439, 415);
         newShell.setText("Link New HDInsight Cluster");
+
     }
 
+    private void refreshContainers(@NotNull ClientStorageAccount storageAccount) {
+    	try {
+        	containersComboBox.removeAll();
+
+			StorageClientSDKManager.getManager().getBlobContainers(storageAccount.getConnectionString())
+					.forEach(blob -> containersComboBox.add(blob.getName()));
+			
+			// not find setMaximumRowCount(*) method in SWT
+		} catch (AzureCmdException e) {
+			setErrorMessage(e.getMessage());
```

Comment:
What's the error message if the account/key is wrong?

Reply:
I test this case and error message is "Error retrieving the Blob Container List".

Reply:
The container `empty check` is already done in Line 291.